src/util/event_loop_signal.hpp

```diff
@@ -27,11 +27,17 @@
 #define REALM_USE_UV 0
 #endif
 
+#if !defined(REALM_USE_CF) && REALM_PLATFORM_APPLE
+#define REALM_USE_CF 1
+#elif !defined(REALM_USE_ALOOPER) && REALM_ANDROID
+#define REALM_USE_ALOOPER 1
+#endif
```

Comment:
I take it that the intent is that .NET will define REALM_USE_CF and REALM_USE_ALOOPER to 0?